src/app/models/User.ts

```diff
@@ -92,10 +92,24 @@ export class User extends AbstractModel implements IUser {
   @BawDateTime()
   public readonly lastSeenAt?: DateTimeTimezone;
 
+  public static get deletedUser(): User {
+    return new User({
+      id: -1,
+      email: "deleted-user@noreply.com.au",
+      userName: "Deleted User",
+    });
+  }
+
+  public static get unknownUser(): User {
+    return new User({
+      id: -2,
+      email: "unknown-user@noreply.com.au",
+      userName: "Unknown User",
+    });
+  }
+
```

Comment:
these users should be constant, and frozen. Then they can be compared via reference equality.

Unless there's a specific need to return a different instance each time?

Reply:
My reasoning at the time was that it would prevent changes to the User object. However `Object.freeze()` fixes that

Reply:
As it turns out, we cannot freeze these objects. The decorator which modifies the user image urls occurs during runtime and will fail if the user object is frozen.